pkg/ui/workspaces/cluster-ui/src/statementDetails/planDetails/plansTable.tsx

```diff
@@ -304,13 +303,7 @@ export function formatIndexes(indexes: string[], database: string): ReactNode {
     return (
       <span key={table}>
         {newLine}
-        <Link
-          className={cx("bold-link")}
-          to={EncodeDatabaseTableUri(database, table)}
-        >
-          {table}
-        </Link>
-        : {indexesList}
+        {table}: {indexesList}
```

Comment:
note: I recently fixed [indexStatsLink](https://github.com/kyle-a-wong/cockroach/commit/2e787c02c0d399c575ae6dde5c7a75723f4ba7fc#diff-aeaaed9042a68111b65dbbebee65766467dda470132fc08cb530b72faeed538fL25) that generates the wrong link for CC console, and looking at the `indexesList` links, it looks like that is wrong as well (I think this is true for all cluster versions, not 24.3, so this isn't a regression due to the db pages work). I created this issue: https://github.com/cockroachdb/cockroach/issues/136476

Reply:
Ack - thanks for filing! I can address that in a follow-up.